### PR TITLE
Created standard page template

### PR DESCRIPTION
**Pull request by tomflenniken**
I've made a few structural changes to the site that I think will improve things overall. Changes can be seen on my pages site deployed here: https://tomflenniken.github.io/

1. Main page scrolls to some informational cards that will have some highlights and promotional copy. (I'll need to work with folks to choose topics and to get the actual content to put here.)
2. External links moved to footer. This frees up site navigation and makes it clearer when you will be leaving the site.
3. Content will need to be updated on the new pages, this will also require working with some others on getting content/links.

### CODE REVIEW

**Comment by tomflenniken** [comment]
Looks like I'm having some issues with the routing when deployed to github pages. I'll take a look at fixing this tomorrow.

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -24,11 +25,12 @@ function Header() {
                 alignItems: 'center',
                 height: '100%',
             }}>
-                <li><a href="https://cortex-command-community.github.io/"><span>Home</span></a></li>
-                <li><a href="https://github.com/cortex-command-community"><span>Downloads</span></a></li>
-                <li><a href="http://wiki.datarealms.com/Cortex_Command"><span>Wiki</span></a></li>
-                <li><a href="https://github.com/cortex-command-community/Cortex-Command-Community-Project-Source/issues"><span>Issues</span></a></li>
-                <li><a href="http://forums.datarealms.com/"><span>Fan Forums</span></a></li>
+                <li><a href="./"><span>Home</span></a></li>
+                <li><a href="./downloads"><span>Downloads</span></a></li>
```
Comment (garethyr) [inline src/components/Header/Header.js]: At least for the time being, if you haven't finished with the downloads page or whatever, you can point the download link to https://github.com/cortex-command-community/Cortex-Command-Community-Project-Data/releases

The rest of the content will need some figuring out, as you said.

**Review by garethyr — APPROVED** [review]
Looks great already. One minor comment for ya.

Also, I'm sure it's a work in progress, but I don't love the grey of the footer background. I get you wanted to differentiate it, but it looks generally less striking than the rest of the page. No need to change it in this PR or anything.

File: src/components/Static/Downloads/Downloads.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+function Downloads() {
+  return (
+    <Template>
+      <h1 style={{ marginTop: '48px' }}>Downloads</h1>
+      <h2>Project Releases</h2>
+      <Card title={'Cortex Command Community Project'}>
+        <p style={{
+          marginBottom: '18px',
+        }}>This is where you can find the community's plans for Cortex Command's future. There'll be rebalancing, new weapons and units, and much more</p>
```
Comment (garethyr) [inline src/components/Static/Downloads/Downloads.js]: Minor typo - there should be a period at the end. All the copy will probably be retouched at some point (though honestly, it's perfectly fine as is), so it's not a big deal, but I noticed it and figured I should mention it.

File: src/components/Header/Header.js (hunk 1 above)
Comment (tomflenniken) [reply src/components/Header/Header.js]: Ah thanks

**Pull request merged**